src/web/mjs/connectors/templates/TakeShobo.mjs

```diff
@@ -9,7 +9,7 @@ export default class TakeShobo extends SpeedBinb {
         super.label = undefined;
         this.tags = [];
         this.url = undefined;
-        this.path = '/';
+        this.path = '/parts_sub_lineup.html';
```

Comment:
Are `StoriaDash` and `WebcomicGamma` also working with the changed path?

Reply:
yup